editor/tabbed_editor.go

```diff
@@ -139,6 +141,20 @@ func (e *TabbedEditor) CloseCurrentEditor() (name string, editor *CodeEditor) {
 	return "", nil
 }
 
+func (e *TabbedEditor) SaveAll() {
+
+	for name, editor := range e.editors {
+		f, err := os.Create(name)
+		if err != nil {
+			log.Printf("Could not save %s : %s", name, err)
+		}
+		defer f.Close()
+		if _, err := f.WriteString(editor.Text()); err != nil {
+			log.Printf("Could not write to file %s: %s", name, err)
+		}
+	}
+}
+
```

Comment:
yes, I thought about this, but not found way for easy traversal of inner editors.